api.planx.uk/modules/send/utils/helpers.test.ts

```diff
@@ -0,0 +1,39 @@
+import type { Passport } from "@opensystemslab/planx-core/types";
+
+import { isApplicationTypeSupported } from "./helpers.js";
+
+vi.mock("@opensystemslab/planx-core", () => {
+  return {
+    getValidSchemaValues: vi
+      .fn()
+      .mockImplementation(() => ["ldc", "ldc.p", "ldc.e", "pp", "pa"]),
+  };
+});
```

Comment:
These mocks resolve questions raised here: https://opensystemslab.slack.com/archives/C01E3AC0C03/p1737996349338569

When testing the route via `supertest`, then `getValidSchemaValues` and planx-core will be called "for real" and not require a mock (eg see `modules/flows/validate/validate.test.ts` >> `describe("ODP Schema file type validation on diff"`), but when simply unit testing like here we need to mock & include partial, realistic `mockImplementation` return values.